foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -161,31 +162,42 @@ public static AbstractConfiguration convertEnvVariable(AbstractConfiguration sou
     return source;
   }
 
-  //inject a copy of servicecomb.xxx for cse.xxx
+  //主要是为了兼容老的版本
```

Comment:
Use en comments when possible

Reply:
If we change this line is better？ 

     duplicateCseConfigToServicecomb(config,
         new DynamicConfiguration(	         new DynamicConfiguration(
             new MicroserviceConfigurationSource(configModelList), new NeverStartPollingScheduler()),	             new MicroserviceConfigurationSource(configModelList), new NeverStartPollingScheduler()),
         "configFromYamlFile");	         "configFromYamlFile");

here do not call duplicateCseConfigToServicecomb and other code remains not changed.